### PR TITLE
Feature: Could not get a single message easily

### PR DESCRIPTION
**Pull request by hoh**
A user could not easily get a single message from the API server, with a clear error message when failing.

Solution: Add function `get_message` singular.

### CODE REVIEW

File: src/aleph_client/asynchronous.py [hunk 1]
```diff
@@ -610,6 +612,33 @@ async def get_messages(
...
+
+async def get_message(
+    item_hash: str,
+    message_type: Type[GenericMessage] = AlephMessage,
+    channel: Optional[str] = None,
+    session: Optional[ClientSession] = None,
+    api_server: str = settings.API_HOST,
+) -> GenericMessage:
+    """Get a single message from its `item_hash`."""
+    raw_program_message = await get_messages(
```
Comment (odesenfans) [inline src/aleph_client/asynchronous.py:626]: Why is it named `raw_program_message`?

File: src/aleph_client/asynchronous.py [hunk 2]
```diff
@@ -610,6 +612,33 @@ async def get_messages(
...
+        hashes=[item_hash],
+        session=session,
+        channels=[channel] if channel else None,
+        api_server=api_server,
+    )
+    if len(raw_program_message.messages) < 1:
+        raise MessageNotFoundError(f"No such hash {item_hash}")
+    assert (
```
Comment (odesenfans) [inline src/aleph_client/asynchronous.py:634]: `assert` can be deactivated, I see your point but a proper exception would be better IMO.

```python
messages = raw_program_message.messages
if not messages:
  raise MessageNotFoundError(f"No such hash {item_hash}")
elif len(messages > 2):
  raise TooManyResults("API returned more than one message, which is unexpected.")
```

File: src/aleph_client/asynchronous.py [hunk 3]
```diff
@@ -610,6 +612,33 @@ async def get_messages(
...
+        len(raw_program_message.messages) == 1
+    ), "Maximum one message should be returned"
+    message: GenericMessage = raw_program_message.messages[0]
+    assert isinstance(message, message_type)
```
Comment (odesenfans) [inline src/aleph_client/asynchronous.py:638]: Same here, not a big fan of asserts.

**Review by odesenfans — COMMENTED** [review]
A few comments.

File: src/aleph_client/asynchronous.py (hunk 2 above)
Comment (hoh) [reply src/aleph_client/asynchronous.py]: Changed.

**Pull request merged**